RELEASING.md

```diff
@@ -259,36 +276,33 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
+1. Merge the PR. This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Checkout the master branch
+## Adding the previous release to docs.projectcalico.org/archive
 
-1. In [netlify.toml](netlify.toml):
-
-   1. Set the `CANDIDATE_RELEASE` environment variable back to an empty string.
-
-   1. Update the `CURRENT_RELEASE` environment variable.
+1. Enable branch deployments for previous release branch on netlify by adding branch name to `Deploy contexts` in site settings.
+ 
+1. Checkout to previous release branch. Add a commit to the branch, an empty commit if necessary (to trigger the branch deploy). 
 
-1. In [netlify/_redirects](netlify/_redirects), add a new line for the new release.
+1. Open a pull requests to upstream previous release branch, get it reviewed and merged. This triggers branch deployment for the previous release.
 
-1. Commit your changes. For example:
+1. Ensure branch deployment is done by visiting to `<previous-release-branch-name>--calico.netlify.app`
```

Comment:
Should we note that the CSS won't load here but that is OK?

Reply:
right, I'll add this.